Core/Src/bmbUtils.c

```diff
@@ -21,22 +20,21 @@
 /* ========================= LOCAL VARIABLES ========================== */
 /* ==================================================================== */
 
-const uint8_t tableLength = 33;
-const float ntcVoltageArray[33] =
-			{0.830305771, 0.897890409, 0.971038326, 1.049937902, 1.134702713, 1.225347925, 
-             1.32176452, 1.423693119, 1.530700113, 1.642159726, 1.757246009, 1.87493832, 
-             1.994042452, 2.113227584, 2.231077109, 2.34614984, 2.457047059, 2.562480286, 
-             2.661334102, 2.752718092, 2.836002466, 2.910833615, 2.977128647, 3.035051094, 
-             3.08497248, 3.127425683, 3.163055933, 3.192574229, 3.216716439, 3.236209893, 
-             3.251747985, 3.263972436, 3.273462335};
-const float zenerVoltageArray[33] =
-			{1.357903819, 1.367736503, 1.377566917, 1.387395079, 1.397221009, 1.407044725, 
-             1.426685585, 1.436502763, 1.456130695, 1.485556757, 1.505163738, 1.534558873, 
-             1.563935877, 1.603077481, 1.642187888, 1.681267562, 1.730074517, 1.788581153,
-             1.847021367, 1.905396115, 1.963706521, 2.031655975, 2.089832632, 2.147951449, 
-             2.206015937, 2.264030521, 2.302681820, 2.350970976, 2.379932725, 2.408886849, 
-             2.428185878, 2.447482310, 2.466776486};
-const float temperatureArray[33] =
+const float ntcVoltageArray[TABLE_SIZE] =
+			{ 0.830305771, 0.897890409, 0.971038326, 1.049937902, 1.134702713, 1.225347925, 
+        1.32176452, 1.423693119, 1.530700113, 1.642159726, 1.757246009, 1.87493832, 
+        1.994042452, 2.113227584, 2.231077109, 2.34614984, 2.457047059, 2.562480286, 
+        2.661334102, 2.752718092, 2.836002466, 2.910833615, 2.977128647, 3.035051094, 
+        3.08497248, 3.127425683, 3.163055933, 3.192574229, 3.216716439, 3.236209893, 
+        3.251747985, 3.263972436, 3.273462335};
+const float zenerVoltageArray[TABLE_SIZE] =
+			{ 1.357903819, 1.367736503, 1.377566917, 1.387395079, 1.397221009, 1.407044725, 
+        1.426685585, 1.436502763, 1.456130695, 1.485556757, 1.505163738, 1.534558873, 
+        1.563935877, 1.603077481, 1.642187888, 1.681267562, 1.730074517, 1.788581153,
+        1.847021367, 1.905396115, 1.963706521, 2.031655975, 2.089832632, 2.147951449, 
+        2.206015937, 2.264030521, 2.302681820, 2.350970976, 2.379932725, 2.408886849, 
+        2.428185878, 2.447482310, 2.466776486};
+const float temperatureArray[TABLE_SIZE] =
```

Comment:
Do you want me to throw the updated temp lookup tables on this PR (2k2 pullup). Not sure if it is worth its own.